### PR TITLE
Adding donation, bankCard form with processing, number of donations and total MAD

### PR DESCRIPTION
**Pull request by Zerkoun-Fatima**

### CODE REVIEW

**Comment by Zerkoun-Fatima** [comment]
add cardBank should be a popup please !!!!

File: src/main/java/org/mql/crowddonating/business/IUserServices.java [hunk 1]
```diff
@@ -9,4 +10,8 @@
     File saveFile(File file);
     void deleteFile(long id, String path);
     
+    /* *** user *** */
+    Donor getDonorByID(long id);
```
Comment (mouadyou) [inline src/main/java/org/mql/crowddonating/business/IUserServices.java]: Hna id khassha tkoun Id machi ID

**Comment by mouadyou** [comment]
Lmouhim rah 9rit chwiya fl code ktebtlek des remarques.
w aham 7aja daba 7eydi 3lina hadik "don" simiyetha "donation" machi "don".
tafe9na mn louwel koulchi yemchi b english.

File: src/main/java/org/mql/crowddonating/business/IUserServices.java (hunk 1 above)
Comment (Zerkoun-Fatima) [reply src/main/java/org/mql/crowddonating/business/IUserServices.java]: Uii c vrai okeeey

**Pull request merged**